codeguru_profiler_agent/agent_metadata/agent_metadata.py

```diff
@@ -83,7 +83,7 @@ def serialize_to_json(self, sample_weight, duration_ms, cpu_time_seconds,
                     "version": self.agent_info.version
                 },
                 "agentOverhead": {
-                    "memory_usage_mb": memory_usage_mb
+                    "memory_usage_mb": int(memory_usage_mb)
```

Comment:
Should this be `memoryInMB`

Reply:
I just checked the internal schema and that's right, I'll update, thanks for noticing this.